Source/JavaScriptCore/b3/B3Kind.h

```diff
@@ -148,6 +146,35 @@ class Kind {
         ASSERT(hasTraps());
         m_traps = traps;
     }
+
+    static constexpr bool hasSensitiveToNaN(Opcode opcode)
```

Comment:
"has" reads a bit poorly as a prefix here. Unless this is convention, I would go with "isSensitiveToNaN" or "supportsSensitiveToNaN" or the like. (might also be a good place for a comment explaining why these op codes are such).

Reply:
Right now, this is a convention for these flags, like, we have hasTraps and hasIsChill.
For now, I'll just rename this flag to `isSensitiveToNaN`, and renaming this to `hasIsSensitiveToNaN`, aligned to the existing `hasIsChill`.